reel-picker/src/main/java/com/innowisegroup/reelpicker/picker/ui/TimePickerFragment.kt

```diff
@@ -21,30 +24,56 @@ internal class TimePickerFragment : Fragment() {
     private var hours: CustomNumberPicker? = null
     private var minutes: CustomNumberPicker? = null
 
-    var localTime: LocalTime? = null
+    private var localDateTime: LocalDateTime? = null
+    private var minDateTime: LocalDateTime? = null
+    private var maxDateTime: LocalDateTime? = null
```

Comment:
why do we need here LocalDateTime var? We can pass LocalTime exactly from ReelPicker. And there will be no need here to convert it toLocalTime